pages/instructorAssessmentAccess/instructorAssessmentAccess.ejs

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <%- include('../partials/navbar'); %>
-    <div id="content" class="container">
+    <div id="content" class="container-fluid">
```

Comment:
This change makes the page (the main container on the page) fill the window.

This change was made on all instructor pages except:
- the home page (with the list of courses and course instances);
- the question appearance (aka preview) page.

The reason for this change is that there were situations where I wanted as a user to make pages wider - to avoid two rows of tabs, to avoid line wrap in the editor, to see the whole gradebook, etc. Before, pages could only be narrow. Now, pages can be wide or narrow - the user can of course still shrink the window to shrink the page.

I'm sure there are lots of ways to improve look and feel. In this particular example, a pop-out button or whatever that switches between `container` and `container-fluid` might be a good idea. This sort of thing seems best left for another PR.

(If anyone wants to improve the UI/UX of the new editor components or of the whole site, I'm 100% on board.)